src/Centreon/Domain/Gorgone/ConfigurationLoader.php

```diff
@@ -83,7 +83,7 @@ public function getApiPort(): ?int
         if (!$this->isOptionsLoaded) {
             $this->loadConfiguration();
         }
-        return (int) $this->gorgoneParameters[self::GORGONE_API_PORT] ?? null;
+        return $this->gorgoneParameters[self::GORGONE_API_PORT] ?? null;
```

Comment:
```suggestion
        return $this->gorgoneParameters[self::GORGONE_API_PORT]
            ? (int) $this->gorgoneParameters[self::GORGONE_API_PORT]
            : null;
```

Reply:
with php 7 return of funtion, :?int auto cast '1' to 1

Reply:
centreon-bot agrees with me :)
furthermore, if we do that, don't cast `isApiConnectionSecure` to bool, cause we need consistency

Reply:
@loiclau 
Your remark is wrong because we use the strict mode of PHP.
Have you never noticed the **declare(strict_types=1)** command at the beginning of every PHP file?